soc/arm/ene_kb1200/etype.h

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
You might be able to use quite a bit of macros from Zephyr's [util.h](https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/sys/util.h#L326) for this file.